package.json

```diff
@@ -31,7 +31,7 @@
     "lint:js:changed": "LIST=`git diff-index --diff-filter=d --name-only --cached HEAD | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --quiet --config changed.eslintrc.js $LIST; fi",
     "lint:js:changed:fix": "LIST=`git diff-index --diff-filter=d --name-only HEAD | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --fix --quiet $LIST; fi",
     "lint:js:untracked:fix": "LIST=`git ls-files --others --exclude-standard | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --fix --quiet $LIST; fi",
-    "lint:sass": "sass-lint --verbose",
+    "lint:sass": "npx stylelint src/**/*.scss",
```

Comment:
We shouldn't need npx here since stylelint is already included in the package file https://github.com/department-of-veterans-affairs/vets-website/blob/master/package.json#L236 -- it would just cause a delay as the dependency is downloaded again

Reply:
Good catch, thank you!